bitacross-worker/bitacross/core/bc-task-sender/src/lib.rs

```diff
@@ -0,0 +1,114 @@
+#![feature(trait_alias)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(all(feature = "std", feature = "sgx"))]
+compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
+
+// re-export module to properly feature gate sgx and regular std environment
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+pub mod sgx_reexport_prelude {
+	pub use futures_sgx as futures;
+}
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+pub use crate::sgx_reexport_prelude::*;
+
+use codec::{Decode, Encode};
+use futures::channel::oneshot;
+use lazy_static::lazy_static;
+use litentry_primitives::AesRequest;
+use log::*;
+#[cfg(feature = "std")]
+use std::sync::Mutex;
+#[cfg(feature = "sgx")]
+use std::sync::SgxMutex as Mutex;
+use std::{
+	format,
+	string::String,
+	sync::{
+		mpsc::{channel, Receiver, Sender as MpscSender},
+		Arc,
+	},
+	vec::Vec,
+};
+
+// TODO: Request should be changed to AesRequest
```

Comment:
Would be good to have TODOs resolved or created follow up issues 🙏🏼

Reply:
Removed